### PR TITLE
Add text objects

### PR DESCRIPTION
**Pull request by aleokdev**
How didn't we have these?? Major oversight :)

Merge after #229.

### CODE REVIEW

**Comment by bjorn** [comment]
I'd suggest to rebase this PR now.

File: src/objects.rs [hunk 1]
```diff
@@ -333,6 +381,95 @@ impl ObjectData {
...
+                color,
+                bold,
+                italic,
+                underline,
+                strikeout,
+                kerning,
+                halign,
+                valign,
+
+        )
```
Comment (bjorn) [inline src/objects.rs]: Code formatting seems a bit strange here.

File: src/objects.rs [hunk 2]
```diff
@@ -333,6 +381,95 @@ impl ObjectData {
...
+        let font_family = font_family.unwrap_or_else(|| "sans-serif".to_string());
+        let pixel_size = pixel_size.unwrap_or(16);
+        let wrap = if wrap == Some(1) { true } else { false };
+        let color = color.unwrap_or(Color {
+            red: 0,
+            green: 0,
+            blue: 0,
+            alpha: 255,
+        });
+        let bold = if bold == Some(1) { true } else { false };
```
Comment (bjorn) [inline src/objects.rs]: Why do we need `let bold = if bold == Some(1) { true } else { false }`? Could just `let bold = bold == Some(1)` suffice? Or `bold.contains(&1)`, heh.

**Review by bjorn — COMMENTED** [review]
Looks good, just two small comments.

File: src/objects.rs (hunk 2 above)
Comment (aleokdev) [reply src/objects.rs]: Oh lol yeah

**Comment by aleokdev** [comment]
Contains is unstable so this will have to do :)

**Review by bjorn — DISMISSED** [review]

**Comment by bjorn** [comment]
Approved, but don't forget the changelog. :-)

**Review by bjorn — APPROVED** [review]

**Pull request merged**